ddev/src/ddev/cli/release/agent/integrations_changelog.py

```diff
@@ -37,7 +37,7 @@ def integrations_changelog(app: Application, integrations: tuple[str], since: st
 
     # Process all checks if no check is passed
     if not integrations:
-        integrations = [integration.name for integration in app.repo.integrations.iter('all')]
+        integrations = [integration.name for integration in app.repo.integrations.iter_all('all')]
```

Comment:
1. Does this also pick up ddev for example?
2. Does it behave the same if you don't pass that argument?

Reply:
1. Urgh, yes it does... but it should not do anything because ddev is not in the agent_requirements file so it will be filtered out right away. 
2. Nope https://github.com/DataDog/integrations-core/blob/b10e1e0bd7b41cc54f2074cd82851843e881685a/ddev/src/ddev/repo/core.py#L194-L200, if I don't pass anything I only choose the integrations that changed